### PR TITLE
austin.intermediate.fall2.lucio.clint

### PR DESCRIPTION
**Pull request by clucio72**

### CODE REVIEW

File: apps/11gist-blog/script.js [hunk 1]
```diff
@@ -1,5 +1,33 @@
 'use strict';
 
 $(document).ready(function() {
-  // You code here
+    $.ajax('https://api.github.com/users/clucio72/gists', {
+        success: function(gists) {
+            gists.forEach(function(gist) {
+                if (gist.description.indexOf('#post Here is my post') > -1) {
```
Comment (tjefferson08) [inline apps/11gist-blog/script.js]: Hmm this looks REALLY specific. Can you think of something more generic here?

File: apps/11gist-blog/script.js [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+                    var $a = $('<a href="' + gist.url + '">' + gist.description.slice(6) + '</a>');
```
Comment (tjefferson08) [inline apps/11gist-blog/script.js]: OOOO why 6? This is what we call a [magic number](http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)

Not a big deal, but something to learn!

File: apps/11gist-blog/script.js [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+                        event.preventDefault();
+                        var href = $(this).attr('href');
+                        $.ajax(href, {
+                            success: function(gist) {
+                                var content = gist['files']['post.md']['content'];
+                                $('#post Here is my post').html(marked(content));
+                                console.log(gist);
+                                $.ajax(gist.comments_url, {
+                                    success: function(comments) {
+                                        $('#comments').html('');
```
Comment (tjefferson08) [inline apps/11gist-blog/script.js:20]: Nice! Good name choices

**Comment by clucio72** [comment]
Okie Dokie, I'll make a few changes.

Thanks!
it was a pleasure taking your class!

**Pull request closed without merging**